.github/workflows/test-release.yml

```diff
@@ -26,17 +26,17 @@ jobs:
     - name: Poetry installation
       run: poetry install
     - name: Run tests
-      run: scripts/run_tests
+      run: scripts/run_unit_tests
```

Comment:
Temporary change until all the other tests get wrangled again

Reply:
Going to do that in a follow-up PR

Reply:
sounds good